documentation/datastructure.dot

```diff
@@ -0,0 +1,209 @@
+digraph metadata {
+    rankdir="LR";
+
+    // dsp-repo Datatypes
+    Dataset [color=blue];
+    Project [color=blue];
+    Person [color=blue];
+    Organization [color=blue];
+    Grant [color=blue];
+    DataManagementPlan [color=blue];
+
+    // other Datatypes
+    Attribution [color=green];
+    DataDownload [color=green];
+    PostalAddress [color=green];
+    Place [color=green];
+
+    Attribution -> agent;
```

Comment:
What actually agent stands here for?
I think also `Attribution` is missing here `hasRole` - this property should exist in both `Attribution` and `Person`? @danielasubotic could you confirm please?
![image](https://user-images.githubusercontent.com/29251288/99406509-3d421300-28ee-11eb-8fdb-0c5fc2687a15.png)

Reply:
the way I understand it is, that `Attribution` (which we borrow from the w3 prov ontology) consists of two informations: Role (how is the agent involved in the thing?) and Agent (to whom is the attribution? which person or organization is the agent?).  
This clearly is valid, otherwise the CI validation would not be ok; I don't know however, how restrictive this really is.  
We'll find that out once the full test suit exists.

Generally, a Person is just a person and doesn't need to have a role (or can have many roles in many things), but an attribution means that a certain person has done something specific in a context, and therefore has a role.

Does that make sense?

Reply:
Ok, I get it:
`{
  "qualifiedAttribution": {
    "role": "",
    "agent": {
      "address": {},
      "email": ""
      ...
    }
  }
}`
Agent is `Person` type and qualifiedAttribution is `Attribution` type. But then Attribution is missing the `hasRole` on the graph, or am I checking older version? 
![image](https://user-images.githubusercontent.com/29251288/99411481-edfee100-28f3-11eb-8576-e4619f3c22bd.png)

Reply:
yes, I missed that, thanks!

Reply:
Fixed it... but it leads to a new problem, see below.

Reply:
It seems all props need to be unique or we miss something?

Reply:
So this will be resolved in #7 right?